src/status_im2/common/qr_code_viewer/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns status-im2.common.qr-code-viewer.view
+  (:require ["qrcode" :as qr-code-js]
+            [cljs-bean.core :as bean]
+            [reagent.core :as reagent]
+            [status-im2.common.qr-code-viewer.style :as style]
+            [react-native.core :as rn]
+            [react-native.svg :as svg]))
+
+(defn qr-code
```

Comment:
I will remove this file and instead use the qr code from quo2 instead. once it's merged: https://github.com/status-im/status-mobile/issues/15582